fleet/models.py

```diff
@@ -22,17 +21,15 @@ def format_phone_number(self):
         return
 
     def get_name(self):
-        name = 'Station ' + self.station_id
-        return name
+        return f'Station {self.pk}'
```

Comment:
So you can't have station 1000 without having station 1, station 2, ...., station 999. Right?